### PR TITLE
Extract version numbers into ext

### PR DESCRIPTION
**Pull request by jonathan-caryl**
Makes updating SDK versions easier in the future

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Tue Sep 20 19:38:51 CST 2016
+#Tue Mar 21 08:52:17 GMT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (NightlyNexus) [inline gradle/wrapper/gradle-wrapper.properties]: 3.4.1 is the latest.

Comment (jonathan-caryl) [inline gradle/wrapper/gradle-wrapper.properties]: But Android Studio is only updating to `3.3` at the moment, presumably it'll move to `3.4.1` some time in the future.

**Comment by IgorGanapolsky** [comment]
@jonathan-caryl There is no reason at all you cannot use the following in your **gradle.properties**:
`distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-all.zip`

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (jonathan-caryl) [inline gradle/wrapper/gradle-wrapper.properties]: I've opened #276 to update the wrapper, then I can rebase this PR on top.

**Comment by jonathan-caryl** [comment]
#276 committed, this one rebased on top

**Pull request merged**